WebContent/js/add/configuration.js

```diff
@@ -89,11 +84,11 @@ function attachFormValidation() {
 		rules: {
 			saveConfigName: {
 				required: true,
-				regex: getPrimNameRegex()
+				regex: "DUMMY REGEX"
 			},
 			saveConfigDesc: {
 				maxlength: $("#saveConfigDesc").attr("maxlength"),
-				regex: getPrimDescRegex()
+				jsregex: getPrimDescRegex()

```

Comment:
Andy realized in dev meeting that jsregex does not exist any more, so this place and below need to be modified.

```suggestion
				regex: getPrimDescRegex()
```